.github/workflows/developer.yml

```diff
@@ -69,7 +62,6 @@ jobs:
             FLAGS=${{ matrix.flags }}
           platforms: linux/${{ matrix.architecture }}
           tags: |
-            ghcr.io/dealii/dealii:master-${{ matrix.ubuntu_version }}-${{ matrix.architecture }}
```

Comment:
`ghcr.io/${{ github.repository }}` instead of `ghcr.io/dealii/` should fix the issue